docs/pages/api_reference/server.mdx

```diff
@@ -248,7 +248,7 @@ or throws an error.
 
 <h3 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-2xl">Defined in</h3>
 
-[src/server/implementation.ts:1344](https://github.com/get-convex/convex-auth/blob/main/src/server/implementation.ts#L1344)
+[src/server/implementation.ts:1350](https://github.com/get-convex/convex-auth/blob/main/src/server/implementation.ts#L1350)
```

Comment:
out of curiosity - are the line numbers here being updated automatically? I'd love to get helpers docs looking good like auth / ents, and want to know any tips & tricks

Reply:
Yeah, this is from `typedoc` and `typedoc-plugin-markdown`, which we use in main Convex docs as well.